dask_ml/model_selection/_search.py

```diff
@@ -120,7 +134,68 @@ def build_graph(
     cache_cv=True,
     multimetric=False,
 ):
+    # This is provided for compatibility with TPOT. Remove
+    # once TPOT is updated and requires a dask-ml>=0.13.0
+    def decompress_params(fields, params):
```

Comment:
This corresponds to gathering cv results as they finished:
https://github.com/dask/dask-ml/pull/433